Useful tools & utilities/Logging.md

```diff
@@ -25,7 +25,7 @@ Timber.w(...)
 Timber.wtf(...)
 ```
 
-For more information about methods, see the javadoc [here](http://jakewharton.github.io/timber).
+For more information about methods, see the javadoc [here](https://jakewharton.github.io/timber/docs/5.x/index.html).
```

Comment:
This is version specific link, but only such links will work

Reply:
Confirmed 😢 but hey, better to have something than to not have a link at all